public/content/wrapped-ether/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: What is Wrapped Ether (wETH)
+description: An introduction to Wrapped Ether (wETH)—an ERC20-compatible version of Ether (ETH). 
+lang: en
+---
+
+# Wrapped Ether (wETH)
+
+[Ether](/eth) (ETH) is the native token on Ethereum, and is used for staking, as a currency and paying for gas fees for computation. However, ETH has limitations in how it can interact with assets and applications built on top of Ethereum, like ERC-20 tokens. In order for ETH to be able to interact with ERC-20 tokens, it must conform with the ERC-20 standard.
```

Comment:
```suggestion
[Ether](/eth) (ETH) is the native token on Ethereum, and is used for staking, as a currency and paying for gas fees for computation. However, ETH has limitations in how it can interact with assets and applications built on top of Ethereum. In order for ETH to be able to interact with ERC-20 tokens, it must conform with the ERC-20 standard.
```